cmd/juju/crossmodel/list.go

```diff
@@ -21,13 +21,29 @@ import (
 )
 
 const listCommandDoc = `
-List information about applications' endpoints that have been shared.
+List information about applications' endpoints that have been shared and who is connected.
+
+The default tabular output shows each user connected (relating to) the offer, and the 
+relation id of the relation. If no relations have been made, no rows are printed.
+
+The summary output shows one row per offer, with a count of active/total relations.
+
+The YAML output shows additional information about the source of connections, including
+the source model UUID.
+
+The output can be filtered by:
+ - interface: the interface name of the endpoint
+ - application: the name of the offered application
+ - user: the name of a connected user
+ - consumer: the name of a user allowed to consume the offer
 
 Examples:
     $ juju offers
     $ juju offers -m model
     $ juju offers --interface db2
     $ juju offers --application mysql
+    $ juju offers --user connected-user
+    $ juju offers --consumer allowed-user
```

Comment:
This seems backwards in my head. 'connected-user' seems to be 'consumer' (who is actively consuming it, vs who is allowed to connect to it).

It seems like there would also be a case for filtering on who is offering the endpoints, which could cause confusion around --user.

Regardless, *I* certainly matched them the other way around.

Reply:
Ok, if you're confused then clearly something needs to be fixed. The intent is to allow filtering by 
- who is connected now to the offer (ie all offers with active or suspended relations made by that user)
- who has permission to consumer the offer

I tried to keep the CLI args to one word. I considered something like:
$ juju offers --allowed-user fred
$ juju offers --connected-user mary

But thought those arg names were getting a bit long. But maybe not if it aids clarity.